joomla/joomla-cms
# [4.0] Modify display and tip for Edit Article in Frontend

### Summary of Changes
Since Email to a friend and Print article have been taken off from J4, we do not need the anymore the Cog and dropdown to choose among actions. We now need only the Edit Article when authorised to do so.
Use of `role="tooltip"]` instead of bootstrap tips.
Cassiopea `[role="tooltip"]` is modified for the look. Placement of tip specific to the article edit id.

Should also be OK for a11y. TO CHECK please as I'm not a specialist.

### Testing Instructions
3 situations:
1. Edit Article when article published // simplified tip
2. Edit Article when article is unpublished // simplified tip
3. Article is checked out by another user

In each case, hover the icon to see the tip.

patch, run NPM


### Actual result BEFORE applying this Pull Request
<img width="1221" alt="Screen Shot 2020-09-07 at 18 35 48" src="https://user-images.githubusercontent.com/869724/92407005-bcbaa580-f139-11ea-8aa7-d5cb92e926c5.png">

<img width="1208" alt="Screen Shot 2020-09-07 at 18 37 57" src="https://user-images.githubusercontent.com/869724/92407046-d8be4700-f139-11ea-9823-f10e47ae7536.png">

<img width="1213" alt="Screen Shot 2020-09-07 at 18 37 23" src="https://user-images.githubusercontent.com/869724/92407066-e1168200-f139-11ea-8b98-22f3143e191b.png">

### Expected result AFTER applying this Pull Request
<img width="866" alt="Screen Shot 2020-09-07 at 18 15 40" src="https://user-images.githubusercontent.com/869724/92407237-4a969080-f13a-11ea-92f1-0c4b989feb9a.png">

<img width="883" alt="Screen Shot 2020-09-07 at 18 14 48" src="https://user-images.githubusercontent.com/869724/92407284-669a3200-f13a-11ea-9f8a-9a2bf6b688fc.png">

<img width="905" alt="Screen Shot 2020-09-07 at 18 13 31" src="https://user-images.githubusercontent.com/869724/92407298-744fb780-f13a-11ea-82d9-bf7954eb213c.png">





### Note
If this is OK, will do for Contact
